config/webpack.prod.js

```diff
@@ -0,0 +1,82 @@
+const merge = require('webpack-merge');
+const Dotenv = require('dotenv-webpack');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+const HtmlWebPackPlugin = require('html-webpack-plugin');
+const CompressionPlugin = require('compression-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
+const commonConfig = require('./webpack.common');
+
+module.exports = merge(commonConfig, {
+  bail: true,
+  mode: 'production',
+  devtool: 'source-map',
+
+  plugins: [
+    new CleanWebpackPlugin(),
+    new Dotenv({
+      path: './.env.production',
+    }),
+    new HtmlWebPackPlugin({
```

Comment:
I think there are lots of options here that aren't necessary, and move this to the `common` webpack config

```
new HtmlWebpackPlugin({
      title: 'Authors Haven',
      filename: 'index.html',
      template: './src/index.html',
      inject: true,
      minify: {
        minifyCSS: true,
        minifyJS: true,
        removeComments: true
      },
      cache: true
    }),
```